### PR TITLE
Documentation

### PR DESCRIPTION
**Pull request by Megha-Joshi**
Screenshot (32)
Screenshot (30)
Screenshot (31)
This is my landing page, introduction page and color-palette page. Please review it.

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+    "liveServer.settings.port": 5501
+}
```
Comment (zoyatanweer) [inline .vscode/settings.json:3]: you can delete this file or put in .gitignore

**Review by zoyatanweer — APPROVED** [review]
Looks great, good to merge.

File: css/nav.css [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+    height: 12.5rem;
+    list-style: none;
+    gap: 1.5rem;
+    justify-content: end;
+    margin-right: 6.25rem;
+    flex-wrap: wrap;
+}
+
+.right-nav li a {
+    text-decoration: none;
```
Comment (dev-prabhat) [inline css/nav.css:77]: instead of tag use classes to apply a style to

File: css/doc.css [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+}
+.line{
+    border-bottom: 2px solid #05cecd;
+    padding-top: 1.25rem;
+    min-width: 14.3rem;
+}
+.right-container{
+    margin-right: 6.25rem;
+}
+.title{
```
Comment (dev-prabhat) [inline css/doc.css]: spaces between classes are irregular

File: css/color.css [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+    width: 80%;
+    height: 3rem;
+    justify-content: space-between;
+    align-items: center;
+    padding: 1.25rem;
+}
+.primary{
+    background: var(--primary-color);
+}
+.accent{
```
Comment (dev-prabhat) [inline css/color.css]: same spacing

**Review by dev-prabhat — CHANGES_REQUESTED** [review]
Some suggestions added

**Review by Shubhamkr2610 — APPROVED** [review]
looks good to me

File: css/doc.css [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+    background: none;
+    border-radius: 0rem;
+}
+.list-menu a{
+    text-decoration: none;
+    color: var(--font-color);
+    font-weight: 600;
+}
+.line{
+    border-bottom: 2px solid #05cecd;
```
Comment (zoef-yt) [inline css/doc.css]: use var for colour.

File: css/main.css [hunk 6]
```diff
@@ -0,0 +1,72 @@
+@import url(css\nav.css);
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@900&display=swap');
```
Comment (zoef-yt) [inline css/main.css]: added a few more font weights for ex:300,400,500,700

**Review by zoef-yt — CHANGES_REQUESTED** [review]

File: .vscode/settings.json (hunk 1 above)
Comment (Megha-Joshi) [reply .vscode/settings.json:3]: done

File: css/color.css (hunk 4 above)
Comment (Megha-Joshi) [reply css/color.css]: Done! Please have a look.

File: css/doc.css (hunk 5 above)
Comment (Megha-Joshi) [reply css/doc.css]: I have edited it. Please have a look.

File: css/main.css (hunk 6 above)
Comment (Megha-Joshi) [reply css/main.css]: I have added more font sizes.

File: css/nav.css (hunk 2 above)
Comment (Megha-Joshi) [reply css/nav.css:77]: I have used it only in the index page. Not required at any other place. That's why I directly give property to that tag.

**Review by zoef-yt — APPROVED** [review]
Make sure to resolve conflicts. Good to merge.

**Pull request merged**